Error saving Xari Level: 135 Job: 910
java.sql.BatchUpdateException: Data truncated for column 'locationtype' at row 1
	at com.mysql.jdbc.PreparedStatement.executeBatchSerially(PreparedStatement.java:2018)
	at com.mysql.jdbc.PreparedStatement.executeBatch(PreparedStatement.java:1454)
	at client.MapleCharacter.saveToDB(MapleCharacter.java:4373)
	at net.server.Server.saveAll(Server.java:130)
	at net.server.Server$1.run(Server.java:174)
	at server.TimerManager$LoggingSaveRunnable.run(TimerManager.java:143)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.runAndReset(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)

---------------------------------
Error saving Xari Level: 135 Job: 910
java.sql.BatchUpdateException: Data truncated for column 'locationtype' at row 1
	at com.mysql.jdbc.PreparedStatement.executeBatchSerially(PreparedStatement.java:2018)
	at com.mysql.jdbc.PreparedStatement.executeBatch(PreparedStatement.java:1454)
	at client.MapleCharacter.saveToDB(MapleCharacter.java:4373)
	at net.server.Server.saveAll(Server.java:130)
	at net.server.Server$1.run(Server.java:174)
	at server.TimerManager$LoggingSaveRunnable.run(TimerManager.java:143)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.runAndReset(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)

---------------------------------
Error saving Xari Level: 135 Job: 910
java.sql.BatchUpdateException: Data truncated for column 'locationtype' at row 1
	at com.mysql.jdbc.PreparedStatement.executeBatchSerially(PreparedStatement.java:2018)
	at com.mysql.jdbc.PreparedStatement.executeBatch(PreparedStatement.java:1454)
	at client.MapleCharacter.saveToDB(MapleCharacter.java:4373)
	at net.server.Server.saveAll(Server.java:130)
	at net.server.Server$1.run(Server.java:174)
	at server.TimerManager$LoggingSaveRunnable.run(TimerManager.java:143)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.runAndReset(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)

---------------------------------
Error saving Xari Level: 135 Job: 910
java.sql.BatchUpdateException: Data truncated for column 'locationtype' at row 1
	at com.mysql.jdbc.PreparedStatement.executeBatchSerially(PreparedStatement.java:2018)
	at com.mysql.jdbc.PreparedStatement.executeBatch(PreparedStatement.java:1454)
	at client.MapleCharacter.saveToDB(MapleCharacter.java:4373)
	at net.server.Server.saveAll(Server.java:130)
	at net.server.Server$1.run(Server.java:174)
	at server.TimerManager$LoggingSaveRunnable.run(TimerManager.java:143)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.runAndReset(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)

---------------------------------
Error saving Xari Level: 135 Job: 910
java.sql.BatchUpdateException: Data truncated for column 'locationtype' at row 1
	at com.mysql.jdbc.PreparedStatement.executeBatchSerially(PreparedStatement.java:2018)
	at com.mysql.jdbc.PreparedStatement.executeBatch(PreparedStatement.java:1454)
	at client.MapleCharacter.saveToDB(MapleCharacter.java:4373)
	at net.server.Server.saveAll(Server.java:130)
	at net.server.Server$1.run(Server.java:174)
	at server.TimerManager$LoggingSaveRunnable.run(TimerManager.java:143)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.runAndReset(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)

---------------------------------
Error saving Xari Level: 135 Job: 910
java.sql.BatchUpdateException: Data truncated for column 'locationtype' at row 1
	at com.mysql.jdbc.PreparedStatement.executeBatchSerially(PreparedStatement.java:2018)
	at com.mysql.jdbc.PreparedStatement.executeBatch(PreparedStatement.java:1454)
	at client.MapleCharacter.saveToDB(MapleCharacter.java:4373)
	at net.server.Server.saveAll(Server.java:130)
	at net.server.Server$1.run(Server.java:174)
	at server.TimerManager$LoggingSaveRunnable.run(TimerManager.java:143)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.runAndReset(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)

---------------------------------
Error saving Xari Level: 135 Job: 910
java.sql.BatchUpdateException: Data truncated for column 'locationtype' at row 1
	at com.mysql.jdbc.PreparedStatement.executeBatchSerially(PreparedStatement.java:2018)
	at com.mysql.jdbc.PreparedStatement.executeBatch(PreparedStatement.java:1454)
	at client.MapleCharacter.saveToDB(MapleCharacter.java:4373)
	at net.server.Server.saveAll(Server.java:130)
	at net.server.Server$1.run(Server.java:174)
	at server.TimerManager$LoggingSaveRunnable.run(TimerManager.java:143)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.runAndReset(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)

---------------------------------
Error saving Xari Level: 135 Job: 910
java.sql.BatchUpdateException: Data truncated for column 'locationtype' at row 1
	at com.mysql.jdbc.PreparedStatement.executeBatchSerially(PreparedStatement.java:2018)
	at com.mysql.jdbc.PreparedStatement.executeBatch(PreparedStatement.java:1454)
	at client.MapleCharacter.saveToDB(MapleCharacter.java:4373)
	at net.server.Server.saveAll(Server.java:130)
	at net.server.Server$1.run(Server.java:174)
	at server.TimerManager$LoggingSaveRunnable.run(TimerManager.java:143)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.runAndReset(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)

---------------------------------
Error saving Xari Level: 135 Job: 910
java.sql.BatchUpdateException: Data truncated for column 'locationtype' at row 1
	at com.mysql.jdbc.PreparedStatement.executeBatchSerially(PreparedStatement.java:2018)
	at com.mysql.jdbc.PreparedStatement.executeBatch(PreparedStatement.java:1454)
	at client.MapleCharacter.saveToDB(MapleCharacter.java:4373)
	at net.server.Server.saveAll(Server.java:130)
	at net.server.Server$1.run(Server.java:174)
	at server.TimerManager$LoggingSaveRunnable.run(TimerManager.java:143)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.runAndReset(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)

---------------------------------
Error saving Xari Level: 135 Job: 910
java.sql.BatchUpdateException: Data truncated for column 'locationtype' at row 1
	at com.mysql.jdbc.PreparedStatement.executeBatchSerially(PreparedStatement.java:2018)
	at com.mysql.jdbc.PreparedStatement.executeBatch(PreparedStatement.java:1454)
	at client.MapleCharacter.saveToDB(MapleCharacter.java:4373)
	at net.server.Server.saveAll(Server.java:130)
	at net.server.Server$1.run(Server.java:174)
	at server.TimerManager$LoggingSaveRunnable.run(TimerManager.java:143)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.runAndReset(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)

---------------------------------
Error saving Xari Level: 135 Job: 910
java.sql.BatchUpdateException: Data truncated for column 'locationtype' at row 1
	at com.mysql.jdbc.PreparedStatement.executeBatchSerially(PreparedStatement.java:2018)
	at com.mysql.jdbc.PreparedStatement.executeBatch(PreparedStatement.java:1454)
	at client.MapleCharacter.saveToDB(MapleCharacter.java:4373)
	at net.server.Server.saveAll(Server.java:130)
	at net.server.Server$1.run(Server.java:174)
	at server.TimerManager$LoggingSaveRunnable.run(TimerManager.java:143)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.runAndReset(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)

---------------------------------
Error saving Xari Level: 135 Job: 910
java.sql.BatchUpdateException: Data truncated for column 'locationtype' at row 1
	at com.mysql.jdbc.PreparedStatement.executeBatchSerially(PreparedStatement.java:2018)
	at com.mysql.jdbc.PreparedStatement.executeBatch(PreparedStatement.java:1454)
	at client.MapleCharacter.saveToDB(MapleCharacter.java:4373)
	at net.server.Server.saveAll(Server.java:130)
	at net.server.Server$1.run(Server.java:174)
	at server.TimerManager$LoggingSaveRunnable.run(TimerManager.java:143)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.runAndReset(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)

---------------------------------
Error saving Xari Level: 135 Job: 910
java.sql.BatchUpdateException: Data truncated for column 'locationtype' at row 1
	at com.mysql.jdbc.PreparedStatement.executeBatchSerially(PreparedStatement.java:2018)
	at com.mysql.jdbc.PreparedStatement.executeBatch(PreparedStatement.java:1454)
	at client.MapleCharacter.saveToDB(MapleCharacter.java:4373)
	at net.server.Server.saveAll(Server.java:130)
	at net.server.Server$1.run(Server.java:174)
	at server.TimerManager$LoggingSaveRunnable.run(TimerManager.java:143)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.runAndReset(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)

---------------------------------
Error saving Xari Level: 135 Job: 910
java.sql.BatchUpdateException: Data truncated for column 'locationtype' at row 1
	at com.mysql.jdbc.PreparedStatement.executeBatchSerially(PreparedStatement.java:2018)
	at com.mysql.jdbc.PreparedStatement.executeBatch(PreparedStatement.java:1454)
	at client.MapleCharacter.saveToDB(MapleCharacter.java:4373)
	at net.server.Server.saveAll(Server.java:130)
	at net.server.Server$1.run(Server.java:174)
	at server.TimerManager$LoggingSaveRunnable.run(TimerManager.java:143)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.runAndReset(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)

---------------------------------
Error saving Xari Level: 135 Job: 910
java.sql.BatchUpdateException: Data truncated for column 'locationtype' at row 1
	at com.mysql.jdbc.PreparedStatement.executeBatchSerially(PreparedStatement.java:2018)
	at com.mysql.jdbc.PreparedStatement.executeBatch(PreparedStatement.java:1454)
	at client.MapleCharacter.saveToDB(MapleCharacter.java:4373)
	at net.server.Server.saveAll(Server.java:130)
	at net.server.Server$1.run(Server.java:174)
	at server.TimerManager$LoggingSaveRunnable.run(TimerManager.java:143)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.runAndReset(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)

---------------------------------
Error saving Xari Level: 135 Job: 910
java.sql.BatchUpdateException: Data truncated for column 'locationtype' at row 1
	at com.mysql.jdbc.PreparedStatement.executeBatchSerially(PreparedStatement.java:2018)
	at com.mysql.jdbc.PreparedStatement.executeBatch(PreparedStatement.java:1454)
	at client.MapleCharacter.saveToDB(MapleCharacter.java:4373)
	at net.server.Server.saveAll(Server.java:130)
	at net.server.Server$1.run(Server.java:174)
	at server.TimerManager$LoggingSaveRunnable.run(TimerManager.java:143)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.runAndReset(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)

---------------------------------
Error saving Xari Level: 135 Job: 910
java.sql.BatchUpdateException: Data truncated for column 'locationtype' at row 1
	at com.mysql.jdbc.PreparedStatement.executeBatchSerially(PreparedStatement.java:2018)
	at com.mysql.jdbc.PreparedStatement.executeBatch(PreparedStatement.java:1454)
	at client.MapleCharacter.saveToDB(MapleCharacter.java:4373)
	at net.server.Server.saveAll(Server.java:130)
	at net.server.Server$1.run(Server.java:174)
	at server.TimerManager$LoggingSaveRunnable.run(TimerManager.java:143)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.runAndReset(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)

---------------------------------
Error saving Xari Level: 135 Job: 910
java.sql.BatchUpdateException: Data truncated for column 'locationtype' at row 1
	at com.mysql.jdbc.PreparedStatement.executeBatchSerially(PreparedStatement.java:2018)
	at com.mysql.jdbc.PreparedStatement.executeBatch(PreparedStatement.java:1454)
	at client.MapleCharacter.saveToDB(MapleCharacter.java:4373)
	at net.server.Server.saveAll(Server.java:130)
	at net.server.Server$1.run(Server.java:174)
	at server.TimerManager$LoggingSaveRunnable.run(TimerManager.java:143)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.runAndReset(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)

---------------------------------
Error saving Xari Level: 135 Job: 910
java.sql.BatchUpdateException: Data truncated for column 'locationtype' at row 1
	at com.mysql.jdbc.PreparedStatement.executeBatchSerially(PreparedStatement.java:2018)
	at com.mysql.jdbc.PreparedStatement.executeBatch(PreparedStatement.java:1454)
	at client.MapleCharacter.saveToDB(MapleCharacter.java:4373)
	at net.server.Server.saveAll(Server.java:130)
	at net.server.Server$1.run(Server.java:174)
	at server.TimerManager$LoggingSaveRunnable.run(TimerManager.java:143)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.runAndReset(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)

---------------------------------
